client/src/components/HistoryViewer/HistoryViewerVersionList.js

```diff
@@ -64,21 +69,26 @@ class HistoryViewerVersionList extends PureComponent {
     );
   }
 
+  renderHeader() {
```

Comment:
Could benefit from a doc block to say "Renders a HeadingComponent, unless it has been disabled (such as when viewing two versions in compare mode)" or something